x/mongo/driver/errors.go

```diff
@@ -97,13 +101,28 @@ func (wce WriteCommandError) Retryable() bool {
 	return (*wce.WriteConcernError).Retryable()
 }
 
+// RetryableWrite returns true if the error is retryable for a write operation
+func (wce WriteCommandError) RetryableWrite(wireVersion *description.VersionRange) bool {
```

Comment:
nit: Can we change the name to `Retryable`? It's never going to be a retryable read.

Reply:
done